samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,364 @@
+.. _bluetooth-isochronous-time-synchronization:
+
+Bluetooth: ISO time synchronization
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Bluetooth® ISO time synchronization sample showcases time-synchronized processing of isochronous data.
+
+The sample demonstrates:
+
+ * How to send isochronous data on multiple streams in such a way that all receiving devices can process the data at the same time.
```

Comment:
```suggestion
 * How to send isochronous data on multiple streams in such a way that all receiving devices and the sending device can process the data at the same time.
```

Reply:
Proposed a small edit to the suggestion here